### PR TITLE
Fixed ui not using proper key for links

### LINKED ISSUE #69: UI: wrong name MW attribute - link should be links
Hi,

The MW attribute for links under maintenance is being sent wrongly from the UI to backend: instead of `link` it should be `links` in https://github.com/kytos-ng/maintenance/blob/99e69d5d7725a707b223399ef014437c2ec06a35/ui/k-toolbar/main.kytos#L123

As a consequence, the MW is not created in the UI when only links are provided (see image below)

Screenshot 2023-02-06 at 3 10 10 PM

### PR DESCRIPTION
**Pull request by Ktmi**
Closes #69 

### Summary

Fixes the main UI component so that the proper key is used when setting `links`.

### Local Tests

Embarrassingly enough, I have yet to actually set up a topology with links, which explains why this was missed before. I still have trouble setting up links, so I haven't been able to properly test this.

### CODE REVIEW

**Review by viniarck — COMMENTED** [review]
Looks good to me. Thanks for this patch, @Ktmi. 

Could you also update the changelog? Also are we going to ship this as a patch `2022.3.1`?

If you also need a quick way to have a topology on mininet with links you can you a linear topo for instance (or alternatively get a more complete topology from [e2e helpers topologies](https://github.com/kytos-ng/kytos-end-to-end-tests/blob/master/tests/helpers.py#L189-L196)):

```
❯ sudo mn --controller=remote,ip=127.0.0.1,port=6653 --switch=ovsk,protocols=OpenFlow13 --topo=linear,3
```

**Comment by Ktmi** [comment]
> Looks good to me. Thanks for this patch, @Ktmi.
> 
> Could you also update the changelog? Also are we going to ship this as a patch `2022.3.1`?
> 
> If you also need a quick way to have a topology on mininet with links you can you a linear topo for instance (or alternatively get a more complete topology from [e2e helpers topologies](https://github.com/kytos-ng/kytos-end-to-end-tests/blob/master/tests/helpers.py#L189-L196)):
> 
> ```
> ❯ sudo mn --controller=remote,ip=127.0.0.1,port=6653 --switch=ovsk,protocols=OpenFlow13 --topo=linear,3
> ```

I was able to figure it out. I was already using mininet for the topologies, but I didn't realize I needed to enable each switch and interface.

**Comment by italovalcy** [comment]
Hi @viniarck,

> Looks good to me. Thanks for this patch, @Ktmi.
> 
> Could you also update the changelog? Also are we going to ship this as a patch `2022.3.1`?

IMHO we should ship this into 2022.3 (i.e., 2022.3.1)

**Comment by viniarck** [comment]
> Hi @viniarck,
> 
> > Looks good to me. Thanks for this patch, @Ktmi.
> > Could you also update the changelog? Also are we going to ship this as a patch `2022.3.1`?
> 
> IMHO we should ship this into 2022.3 (i.e., 2022.3.1)

Right. Sounds good. @Ktmi we have info [about how to release a NApp on this page](https://kytos-ng.github.io/procedures/releases.html#releasing-napps), feel free to proceed updating kytos.json in this same PR. If you need help let me know.

**Comment by viniarck** [comment]
> > Looks good to me. Thanks for this patch, @Ktmi.
> > Could you also update the changelog? Also are we going to ship this as a patch `2022.3.1`?
> > If you also need a quick way to have a topology on mininet with links you can you a linear topo for instance (or alternatively get a more complete topology from [e2e helpers topologies](https://github.com/kytos-ng/kytos-end-to-end-tests/blob/master/tests/helpers.py#L189-L196)):
> > ```
> > ❯ sudo mn --controller=remote,ip=127.0.0.1,port=6653 --switch=ovsk,protocols=OpenFlow13 --topo=linear,3
> > ```
> 
> I was able to figure it out. I was already using mininet for the topologies, but I didn't realize I needed to enable each switch and interface.

Great. 

FYI, `kytosd` has a CLI option `-E` to enable all entities by default. That facilitates a lot to avoid having to enable the entities especially when developers are running it.

File: CHANGELOG.rst [hunk 1]
```diff
@@ -25,6 +25,14 @@ Changed
 =======
 
 
+[2022.3.0] - 2023-02-08
```
Comment (viniarck) [inline CHANGELOG.rst:28]:
```suggestion
[2022.3.1] - 2023-02-08
```

**Review by viniarck — APPROVED** [review]
LGTM. Thanks for fixing this, @Ktmi. 

I left a trivial change suggestion in the changelog though, that you probably meant to type. 

img

**Pull request merged**